packages/strapi-design-system/src/Textarea/TextareaInput.js

```diff
@@ -0,0 +1,72 @@
+import React, { forwardRef } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { useField } from '../Field';
+
+const Textarea = styled.textarea`
+  display: block;
+  width: 100%;
+
+  border: 1px solid ${({ theme, hasError }) => (hasError ? theme.colors.danger600 : theme.colors.neutral200)};
+  border-radius: ${({ theme }) => theme.borderRadius};
+
+  padding-left: ${({ theme, hasLeftAction }) => (hasLeftAction ? 0 : theme.spaces[4])};
+  padding-right: ${({ theme, hasRightAction }) => (hasRightAction ? 0 : theme.spaces[4])};
+  padding-top: ${({ theme }) => `${theme.spaces[3]}`};
+  padding-bottom: ${({ theme }) => `${theme.spaces[3]}`};
+
+  font-weight: 400;
+  font-size: ${14 / 16}rem;
+
+  color: ${({ theme, disabled }) => (disabled ? theme.colors.neutral600 : theme.colors.neutral800)};
+  background: ${({ theme, disabled }) => (disabled ? teme.colors.neutral150 : theme.colors.neutral0)};
+
+  /* managed by switching borders */
+  outline: none;
+  &:focus {
+    border: 1px solid ${({ theme }) => theme.colors.primary600};
+  }
+
+  ::placeholder {
+    color: ${({ theme }) => theme.colors.neutral500};
+    opacity: 1;
+  }
+`;
+
+export const TextareaInput = forwardRef(({ disabled, ...props }, ref) => {
+  const { id, error, hint, name } = useField();
+
+  let ariaDescription;
+
+  if (error) {
+    ariaDescription = `field-error-${id}`;
+  } else if (hint) {
+    ariaDescription = `field-hint-${id}`;
+  }
```

Comment:
What's the point of those aria desc `field-error-` and `field-hint-` ?

Reply:
just in case the id is used by another field. "email" might be on used in multiple places, maybe 😊